src/test/resources/test-invalid-xml-conf/conf/config.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+
+    Copyright 2019-present HiveMQ GmbH
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<invalid-config>
```

Comment:
Should we have an xsd, or was this already discussed?

Reply:
I already have the XSD created, but if I add it to `/resources` as usual, there's nothing to move it to the `/conf` folder, so we can either set that up or just add it to the `/conf` directly (which I didn't like because of consistency)

Reply:
@LukasBrand I've pushed the config.xsd to the `/resources` folder, can you take over ensuring it is applied please?(requires changing config.xml in `/conf/examples` as well)